feature: add cookie hint on every html page (DSGVO)

ronnyfriedland opened the pull request.

avaris commented:
Wouldn't you do this in the theme?

Also, looks like this would require `jQuery` but no mention of it in the readme. 

And a defined but never used variable in the code: `JS_LIBRARIES`

ronnyfriedland commented:
Hi,
thank you for giving feedback.

Putting it to the theme would be much easier you are right - but you have to always use the same theme or requires modifying any theme before using.

I removed th unused vaiable and import and added the dependency to jquery.
